docs/cloud/dbaas/web-interface.md

```diff
@@ -13,23 +13,29 @@ In the column on the left hand side you can go to `Database` -> `Instances` and
 6. `Locality` - Is not needed. In the future DBaaS will support clustered databases and at that point anti-affinity should be the preferred option in most cases.
 
 On the next page `Database access`:
 
-1. `Is public` - You need to check the public checkbox.
-2. `Allowed CIDRs` - Here you want to add your allowed IP-addresses in the format `$IP/32` if you want to allow multiple IP-addresses. You need to separate them by a comma `,`.
-3. You don't need to use the `Intialize Databases` tab. It is not recommended to create an admin user.
-4. If you have a backup that you want to restore, you can do it from the `Advanced` tab. Then you choose the `Source for Initial State` -> `Restore from Backup` and then choose `Backup name`
-5. Now you can press `Launch`
-6. Once the database instance has launched. You can press on the name of the database instance.
-7.  From here you can choose the `Database` tab and press `+ Create Database` and fill in the Name and possible collection, and Character Set and press `Create Database`
-8.  Then you can go to the `Users` tab to create a new database user. The database user needs a name and a strong password. It is recommended that you generate a long random password. You can also specify from which Host the user is allowed to access the database instance from. You should also specify to which databases the users should have access to. When you are done you can press `Create user`
-9.  Now you can go to the database specific documentation to find out further instructions on how to use the database:
+7. `Allowed CIDRs` - Here you want to add your allowed IP-addresses in the format `$IP/32` if you want to allow multiple IP-addresses. You need to separate them by a comma `,`.
+
+On the third page `Intialize Databases:
+
+8. In the `Initial Databases` field you can write what databases should be initialized. You can add additional database after the instance have started.
+9. `Initial Admin User` adding first user that you want to use to connect to the database. You can add more user accounts after the database instance have started.
+10. `Password` for the you first user. Please make sure that this password is not used anywhere else.
+11. `Allowed Host (optional)` , this feature is not supported at the moment but in the future it will possible to limit users access based on IP.
+
+On the fourth page `Advanced`:
+
+12. You don't need to do anything but it is possible to launch a new database from a backup. If you choose to launch a database from a backup you don't need to specify `Initial Admin User`, `Passowrd`, or `Initial Databases`
+13. Now you can press `Launch`
+14. Once the database instance has launched. You can press on the name of the database instance and manage additional Users, databases backups a.s.o. but it might take a couple minutes before the instance is up and running.
```

Comment:
```suggestion
14. Once the database instance has launched, you can click the name of the database instance and manage additional items, such as users, backups etc. But it might take a couple minutes before the instance is up and running.
```

Reply:
Updated as suggested. Thanks for the suggestions.